### DISCUSSION TITLE
Chia node not showing any upload or download for peers.

### DISCUSSION THREAD

**Discussion by pancakestacker** [Chia-Network/chia-blockchain#5186] [Support]
So I recently checked my main farming node and noticed t couldn’t access the internet on my browser. Decided to give it a reset and now I’m noticing the peers I’m connected too aren’t showing any upload or download speed. Does anyone know if this affects mining? And if so how can I go about fixing this? Could this be my isp blocking my connections? Please any advice helps!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
